### PR TITLE
Sound component

### PR DESCRIPTION
**Pull request by krapspark**
PTAL anybody.

@BrettBukowski @Ako0 @learningd3

### CODE REVIEW

File: app/components/story/AudioButton.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      >
+        <View>
+          <Svg
+            width={20}
+            height={20}
+            viewBox="0 0 25 25"
+          >
+            <Path
+              fill={this.props.isMenuVisible ? '#C35138' : '#93732E'}
+              d="M3 9v6h4l5 5V4L7 9H3zm13.5 3c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM14 3.23v2.06c2.89.86 5 3.54 5 6.71s-2.11 5.85-5 6.71v2.06c4.01-.91 7-4.49 7-8.77s-2.99-7.86-7-8.77z"
```
Comment (mattam) [inline app/components/story/AudioButton.js]: This looks like a shape constant, and could maybe be extracted out and named so we know what shape it describes?

**Comment by BrettBukowski** [comment]
can you add instructions on how to install / `npm link`?

File: app/components/Story.js [hunk 2]
```diff
@@ -58,10 +60,15 @@ class Story extends Component {
...
 
+    // TODO: use real mp3 URL
     const chapterData = bottomChapter.sections.map((section, index) =>
       ({
         topText: topChapter.sections[index],
         bottomText: bottomChapter.sections[index],
+        topTrackId: `${chapterId}-${index}-top`,
+        topTrackUrl: 'https://ia600706.us.archive.org/4/items/Sample_Audio_Clips_mp3/KuumbaPodcast1.mp3',
+        bottomTrackId: `${chapterId}-${index}-bottom`,
+        bottomTrackUrl: 'https://ia600706.us.archive.org/4/items/Sample_Audio_Clips_mp3/KuumbaPodcast1.mp3',
```
Comment (BrettBukowski) [inline app/components/Story.js]: so should we hold off on merging until the real mp3 data is available?

File: app/components/story/AudioButton.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+import {
+  StyleSheet,
+  View,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+
+const PLAYING = 'PLAYING';
+const PAUSED = 'PAUSED';
```
Comment (BrettBukowski) [inline app/components/story/AudioButton.js]: are these constants used anywhere?

File: app/components/story/AudioMenu.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+function rewindIcon() {
+  return (<Path fill={"#93732E"} d="M12 5V1L7 6l5 5V7c3.31 0 6 2.69 6 6s-2.69 6-6 6-6-2.69-6-6H4c0 4.42 3.58 8 8 8s8-3.58 8-8-3.58-8-8-8z"/>);
+}
+
+function playIcon() {
+  return (<Path fill={"#93732E"} d="M8 5v14l11-7z"/>);
+}
+
+function pauseIcon() {
+  return (<Path fill={"#93732E"} d="M6 19h4V5H6v14zm8-14v14h4V5h-4z"/>);
```
Comment (BrettBukowski) [inline app/components/story/AudioMenu.js]: there's a color constants file where we can put these fill colors in

File: app/components/story/AudioButton.js (hunk 1 above)
Comment (krapspark) [inline app/components/story/AudioButton.js]: kk

File: app/components/Story.js (hunk 2 above)
Comment (krapspark) [inline app/components/Story.js]: yep. not going to merge until we have sound files.

File: app/components/story/AudioButton.js (hunk 3 above)
Comment (krapspark) [inline app/components/story/AudioButton.js]: nope. removed.

**Comment by BrettBukowski** [comment]
https://github.com/basillabs/Mission111/pull/78 incorporated these changes and added the audio files.

**Pull request closed without merging**